cr8/cli.py

```diff
@@ -32,6 +37,17 @@ def lines_from_stdin(default=None):
         yield line
 
 
+def _create_parse():
+    if simdjson:
+        parser = simdjson.Parser()
+
+        def parse(line):
+            doc = parser.parse(line)
+            return OrderedDict(zip(doc.keys(), doc.values()))
```

Comment:
`simdjson.loads()` will perform a bit better than this, since you're loading the entire dict anyways by using keys() and values() instead of the proxy objects.

Since you're not targeting py2, all the dicts simdjson returns are naturally ordered so you don't need OrderedDict either.

Reply:
> Since you're not targeting py2, all the dicts simdjson returns are naturally ordered so you don't need OrderedDict either.

Thanks. Somehow I thought that it's only since Python 3.7, but looks like I it was already the case in Python 3.6 as well. 

So re-using the `parser` and doing a `parser.parse(...).as_dict()` will also be slower than `simdjson.loads()` ?

Reply:
`simdjson.loads()` is functionally the same as `parser.parse(...).as_dict()`, and will work at the same speed. Both are better than creating two separate lists (.keys() and .values()) and then creating a dict from it.